salt/elasticsearch/files/so-elasticsearch-pipelines

```diff
@@ -26,8 +26,8 @@ ELASTICSEARCH_INGEST_PIPELINES="/opt/so/conf/elasticsearch/ingest/"
 echo -n "Waiting for ElasticSearch..."
 COUNT=0
 ELASTICSEARCH_CONNECTED="no"
-while [[ "$COUNT" -le 240 ]]; do
-    {{ ELASTICCURL }} -k --output /dev/null --silent --head --fail -L https://"$ELASTICSEARCH_HOST":"$ELASTICSEARCH_PORT"
+while [[ "$COUNT" -le 10 ]]; do
+    {{ ELASTICCURL }} -k --connection-timeout 60 --output /dev/null --silent --head --fail -L https://"$ELASTICSEARCH_HOST":"$ELASTICSEARCH_PORT"
```

Comment:
The default timeout for the curl command can range from 60 to 120 seconds. Hard code this value instead of relying on the OS to make this determination for us.